src/rollup-plugin-remote-cdn.ts

```diff
@@ -1,10 +1,10 @@
-import mkdirp from 'mkdirp';
-import cacache from 'cacache';
-import url from 'url';
```

Comment:
👍

Reply:
Sidenote: I usually like `strict: true` and `noUnusedImports: true` as TSConfig options. Not sure if either of those will pass at this point, but that’d be a nice cleanup goal someday.

Reply:
That's what I use generally. I would like to see us make it to that at some point.